network.tf

```diff
@@ -53,8 +53,8 @@ resource "google_compute_global_address" "private_ip_address" {
 
   purpose       = "VPC_PEERING"
   address_type  = "INTERNAL"
-  address       = "192.168.4.0"
-  prefix_length = 22
+  address       = split("/",var.database_subnet_cidr)[0]
```

Comment:
From comment: Only used when `create_postgres=true` so this line and 57 should have a guard on them.
```
address = var.create_postgres ? split("/",var.database_subnet_cidr)[0] : null
```

Reply:
the guard is already in the resource:  
count = var.create_postgres ? 1 : 0